tests/support_server/setup.pm

```diff
@@ -334,6 +345,9 @@ sub setup_xdmcp_server {
 
 sub setup_iscsi_server {
     return if $iscsi_server_set;
+    record_info 'iSCSI server setup';
+    zypper_ar('http://openqa.suse.de/assets/repo/fixed/SLE-12-SP3-Server-DVD-x86_64-GM-DVD1', name => 'sles12sp3dvd1_repo');
```

Comment:
Why do we add this repo as a hardcode one?

Reply:
I don't like it also, but unfortunately Support Server is currently based on the exact QCOW OS image which is next to change, but not in this push.

Reply:
Ok, one more comment is, 
`http://openqa.suse.de/assets/repo/fixed/SLE-12-SP3-Server-DVD-x86_64-GM-DVD1` is hardcode for x86_64 only, do we need to support other arches like aarch64/s390x/ppc64le?